### PR TITLE
Replace aliases for functions and append repos without echo

### PR DESCRIPTION
**Pull request by tricantivu**

### CODE REVIEW

**Comment by tricantivu** [comment]
Changes have been made to resolve previous merge conflict.

File: .bashrc [hunk 1]
```diff
@@ -112,6 +112,35 @@ if ! shopt -oq posix; then
...
+
+    mkdir -p ~/.rhino/{updates,config}
+
+    for rhinoFile in ~/.rhino/updates/configuration ~/.rhino/updates/config-v2; do
+
+        : >  "${rhinoFile}"
+    done
+
+    sudo apt update && sudo apt upgrade -y
+}
```
Comment (ajstrongdev) [inline .bashrc]: Should not be in `.bashrc` but rather in `.bash_aliases` as to keep the two files separate when the time comes to update them. Is this possible?

**Review by ajstrongdev — CHANGES_REQUESTED** [review]

File: .bashrc (hunk 1 above)
Comment (tricantivu) [reply .bashrc]: It is possible, however putting functions in an **aliases** file doesn't really make sense.

Or we could create a .bash_functions file and then source it in .bashrc

Comment (ajstrongdev) [reply .bashrc]: The second option seems the most sensible. I wasn't asking to put it into the `.bash_aliases` file more than I was saying to do something similar to that.

Comment (tricantivu) [reply .bashrc]: Ok then, I will create a .bash_functions file and overwrite the commit history.

I am currently out, I'll mention you when I finish.

Comment (ajstrongdev) [reply .bashrc]: Thank you

Comment (tricantivu) [reply .bashrc]: Done @Chadano!

**Pull request closed without merging**